lib/presentation/widget/home/home_viewmodel.dart

```diff
@@ -87,12 +87,16 @@ class HomeViewModel extends BaseViewModel {
     });
   }
 
-  ThunkAction<AppState> _getAuthorizedUserAction() {
-    return (Store<AppState> store) async {
+  void _getAuthorizedUser() {
+    store.dispatch(_getAuthorizedUserAction());
+  }
+
+  OnlineThunkAction _getAuthorizedUserAction() {
+    return OnlineThunkAction((Store<AppState> store) async {
       await _getAuthorizedInteractor.execute().then((result) => result.fold(
-        (left) => store.dispatch(_getAuthorizedUserFailureAction(left)),
+        (left) => null,
```

Comment:
Does it impact to `Login` flow?

Reply:
No it does not. This is just for getting user info after `Login` flow